packages/www/src/pages/index.tsx

```diff
@@ -232,34 +229,33 @@ export default function Homepage() {
       <section class="brisa-section goal">
         <div class="code-example">
           <div class="info">
-            <h2>🤔 Web Platform Framework?</h2>
+            <h2>🤔 What does Web Platform Framework mean?</h2>
             <p>
-              Brisa's mission is to unify the both worlds (server and client)
-              using the Web Platform. One key feature is the ease with which{' '}
-              <b>Web Components</b> can be used, utilizing{' '}
+              Brisa's mission is to unify server and client using the Web
```

Comment:
Mission of a product should be defined by the original author. On this one I think we should respect Aral's original wording. Unless you want to suggest a change on the mission, I would prefer to respect the wording. This is my personal thinking, happy to listen other arguments.

Reply:
I'll let Aral decide on this one. @aralroca , WDYT?

Reply:
It's the same mission but simplified, LGTM.